src/plugins/console/server/lib/spec_definitions/json/generated/delete_by_query_rethrottle.json

```diff
@@ -9,6 +9,6 @@
     "patterns": [
       "_delete_by_query/{task_id}/_rethrottle"
     ],
-    "documentation": "https://www.opensearch.org/guide/en/elasticsearch/reference/current/docs-delete-by-query.html"
+    "documentation": "https://opensearch.org/docs/latest/opensearch/rest-api/document-apis/delete-by-query/"
```

Comment:
I don't see the rethrottle details in the documentation - keep the link but add this to `noDocumentation`.